src/components/nav-tabs/AboutPage.js

```diff
@@ -8,31 +8,61 @@ export function AboutContent() {
         <Container pb={205}>
             <Title order={2} my="md">Our Purpose</Title>
             <Text pb="sm">
-              Open source software is widely available, but it faces an
-              operatations barrier to entry into production environments.
+              Why build an all-Open Source community cloud? How does it help learn
+              things and make things better?
+            </Text>
+            <Text pb="sm">
+              Open Source software is widely available, but it faces an
+              operations-barrier when bringing it to a production environment.
               Proprietary services have attempted to address this barrier, but
-              they undermine the open source model.
+              they undermine the Open Source development model because lessons
+              learned from operating the code are invisible to the Open Source
+              developers.
             </Text>
             <Text pb="sm">
-              To overcome this operations barrier in an open source way, we
-              must switch to an <b><i>open source approach to operations</i></b>.
-              This means that developers and operators collaborate to apply a
+              To overcome this operations barrier in an Open Source way, we
+              must switch to an <b>Open Source-like approach to operations</b>,
+              or <i>Open Operations</i> (<i>Open Ops</i>). Open Operations means
+              developers and operators collaborating Openly to apply a
```

Comment:
Do we also want to include `Open Source Services` and link to https://www.operate-first.cloud/community/open-source-services.html ?